docs/aws.md

```diff
@@ -0,0 +1,86 @@
+# AWS
+
+## Adding new resources
+
+1. Familiarize yourself with the official AWS pricing page for the service as well as the Terraform documentation for the resource you want to add. Note all factors that influence the cost.
+2. Download and familiarize yourself with the pricing data CSV. This can be done by first checking the [index.json](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/index.json), finding the respective service under the `offers` key and downloading the file at the URL under the `currentVersionUrl` (replace `json` with `csv`).
+3. Find the names of all columns that contain relevant cost factors and check that the `aws/field/field.go` file contains them - add them if this is not the case. The constant name should be a correct Go identifier, while the comment should contain the name as it appears in the CSV file.
+4. Run `go generate ./...` to regenerate the field list.
```

Comment:
I had this when I wanted to regenerate the field list : 

![image](https://user-images.githubusercontent.com/23166349/147613767-800e5364-83fb-444b-825e-561ce9437212.png)

Maybe adding somes notes for download/installing this [package](https://pkg.go.dev/github.com/dmarkham/enumer#section-readme) ?

Reply:
Changed it to use `make generate` which actually takes care of it.